### ISSUE TITLE
Broken Build, RAC 8.0.0: tvOS; ReactiveMapKit

### ISSUE THREAD

**Issue by MikeTeeVee** [ReactiveCocoa/ReactiveCocoa#3619]
Anyone else not able to build RAC for tvOS? iOS platform is building fine.

Tools: Carthage 0.30.1; Xcode 10.0 (10A255)

Issue:

With one-line Cartfile,

```
github "ReactiveCocoa/ReactiveCocoa" == 8.0.0
```
This build command does not succeed:
```
carthage update --platform tvos 
```
Again, iOS build is not broken.

Investigation:

- ReactiveMapKit.scheme's `BlueprintIdentifier` refers to ReactiveMapKit native target object in project.pbxproj (id = 9AC03A561F7CC3BF00EC33C1). 
- ReactiveMapKit native target object's `Frameworks` build phase refers to ReactiveCocoa.framework build file object (id = 9AC03A641F7CC6B300EC33C1), which resolves to file reference object (id = D04725EA19E49ED7006002AA). 
- This file reference object is the product reference (framework build artifact) for ReactiveCocoa-**macOS** native target object (id = D04725E919E49ED7006002AA).

In other words, the ReactiveMapKit target imports the macOS ReactiveCocoa.framework. Why iOS builds and tvOS does not is a mystery, but the build errors suggest that the non-tvOS code is causing the build to fail. 

This suggests that ReactiveMapKit scheme needs to be split up into multiple schemes and targets separated for each Apple OS (ReactiveMapKit-iOS, ReactiveMapKit-tvOS, etc).

**Comment by andersio** [comment]
Xcode used to build universal framework targets fine this way, and it appears to be continued to work when building via Xcode 10 GUI. I will try to poke around `xcodebuild` to see if there is any potential solution.

**Comment by mdiep** [comment]
> Xcode used to build universal framework targets fine this way

There's always been issues at times. We've often had to disable implicit dependencies to work around issues with universal framework targets.
